src/main/resources/application.yaml

```diff
@@ -81,8 +81,9 @@ bullet:
     # The list of responders that extend the com.yahoo.bullet.pubsub.PubSubResponder class. Anything listed here will
     # be instantiated  and anything returning from the Bullet backend will be passed to it
     responders:
```

Comment:
Can we change this to responder then?

Reply:
because responderClasses -> responder.classes? makes sense